ccl/changefeedccl: TestChangefeedSchemaChangeAllowBackfill_Legacy failed

ccl/changefeedccl.TestChangefeedSchemaChangeAllowBackfill_Legacy [failed](https://mesolite.cluster.engflow.com/invocations/default/b2c9e931-9b7a-43b2-8d09-f5265fa860b5?testReportRun=1&testReportShard=15&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [acfdcc8e2e5da1d4d37fee05f41a067e855de912](https://github.com/cockroachdb/cockroach/commits/acfdcc8e2e5da1d4d37fee05f41a067e855de912):


```
=== RUN   TestChangefeedSchemaChangeAllowBackfill_Legacy
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedSchemaChangeAllowBackfill_Legacy507446872
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:2995: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedSchemaChangeAllowBackfill_Legacy507446872
--- FAIL: TestChangefeedSchemaChangeAllowBackfill_Legacy (139.67s)
=== RUN   TestChangefeedSchemaChangeAllowBackfill_Legacy/cloudstorage/drop_column
    changefeed_test.go:2906: 
        	Error Trace:	pkg/ccl/changefeedccl/helpers_test.go:268
        	            				pkg/ccl/changefeedccl/helpers_test.go:397
        	            				pkg/ccl/changefeedccl/changefeed_test.go:2906
        	Error:      	Received unexpected error:
        	            	operation "withTimeout-1066080028244246529" timed out after 30s (given timeout 30s): stopping job due to TestFeed timeout
        	            	(1) operation "withTimeout-1066080028244246529" timed out after 30s (given timeout 30s)
        	            	Wraps: (2) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.stopFeedWhenDone.func3
        	            	  | 	pkg/ccl/changefeedccl/testfeed_test.go:2974
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.stopFeedWhenDone.func1
        	            	  | 	pkg/ccl/changefeedccl/testfeed_test.go:2963
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1700
        	            	Wraps: (3) stopping job due to TestFeed timeout
        	            	Error types: (1) *timeutil.TimeoutError (2) *withstack.withStack (3) *errutil.leafError
        	Test:       	TestChangefeedSchemaChangeAllowBackfill_Legacy/cloudstorage/drop_column
--- FAIL: TestChangefeedSchemaChangeAllowBackfill_Legacy/cloudstorage/drop_column (30.23s)
=== RUN   TestChangefeedSchemaChangeAllowBackfill_Legacy/cloudstorage
    helpers_test.go:1040: making server as secondary tenant
    helpers_test.go:1119: making cloudstorage feed factory
    changefeed_test.go:2812: using legacy schema changer
--- FAIL: TestChangefeedSchemaChangeAllowBackfill_Legacy/cloudstorage (138.71s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=15</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeAllowBackfill_Legacy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-49620

ccl/changefeedccl.TestChangefeedSchemaChangeAllowBackfill_Legacy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19510583?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19510583?buildTab=artifacts#/) on master @ [acfdcc8e2e5da1d4d37fee05f41a067e855de912](https://github.com/cockroachdb/cockroach/commits/acfdcc8e2e5da1d4d37fee05f41a067e855de912):


```
=== RUN   TestChangefeedSchemaChangeAllowBackfill_Legacy
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/a2df272b68db5af56229d82d0d6ebe3f/logTestChangefeedSchemaChangeAllowBackfill_Legacy559826787
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:2995: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/a2df272b68db5af56229d82d0d6ebe3f/logTestChangefeedSchemaChangeAllowBackfill_Legacy559826787
--- FAIL: TestChangefeedSchemaChangeAllowBackfill_Legacy (129.93s)
=== RUN   TestChangefeedSchemaChangeAllowBackfill_Legacy/enterprise
    helpers_test.go:1040: making server as secondary tenant
    helpers_test.go:1119: making enterprise feed factory
    helpers_test.go:1133: pgURL enterprise root
    changefeed_test.go:2812: using legacy schema changer
--- FAIL: TestChangefeedSchemaChangeAllowBackfill_Legacy/enterprise (129.88s)
=== RUN   TestChangefeedSchemaChangeAllowBackfill_Legacy/enterprise/drop_column
    changefeed_test.go:2906: 
        	Error Trace:	pkg/ccl/changefeedccl/helpers_test.go:268
        	            				pkg/ccl/changefeedccl/helpers_test.go:397
        	            				pkg/ccl/changefeedccl/changefeed_test.go:2906
        	Error:      	Received unexpected error:
        	            	operation "withTimeout-1066093735390216193" timed out after 30s (given timeout 30s): stopping job due to TestFeed timeout
        	            	(1) operation "withTimeout-1066093735390216193" timed out after 30s (given timeout 30s)
        	            	Wraps: (2) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.stopFeedWhenDone.func3
        	            	  | 	pkg/ccl/changefeedccl/testfeed_test.go:2974
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.stopFeedWhenDone.func1
        	            	  | 	pkg/ccl/changefeedccl/testfeed_test.go:2963
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_arm64.s:1223
        	            	Wraps: (3) stopping job due to TestFeed timeout
        	            	Error types: (1) *timeutil.TimeoutError (2) *withstack.withStack (3) *errutil.leafError
        	Test:       	TestChangefeedSchemaChangeAllowBackfill_Legacy/enterprise/drop_column
--- FAIL: TestChangefeedSchemaChangeAllowBackfill_Legacy/enterprise/drop_column (31.76s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeAllowBackfill_Legacy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/changefeedbase/settings.go
+++ b/pkg/ccl/changefeedccl/changefeedbase/settings.go
@@ -346,5 +346,5 @@ var Quantize = settings.RegisterDurationSettingWithExplicitUnit(
 	settings.ApplicationLevel,
 	"changefeed.resolved_timestamp.granularity",
 	"the granularity at which changefeed progress is quantized to make tracking more efficient",
-	time.Duration(metamorphic.ConstantWithTestRange("changefeed.resolved_timestamp.granularity", 0, 0, 20))*time.Second,
+	time.Duration(metamorphic.ConstantWithTestRange("changefeed.resolved_timestamp.granularity", 0, 0, 10))*time.Second,
 )

